docs/src/tutorials/AtomsBase_interface.md

```diff
@@ -0,0 +1,130 @@
+# AtomsBase Interface
+
+ACEpotentials has a new [AtomsBase](https://github.com/JuliaMolSim/AtomsBase.jl) based interface in addition to [JuLIP](https://github.com/JuliaMolSim/JuLIP.jl) based interface.
+
+AtomsBase allows easy communication between different Julia programs. In the future this interface will became the default one and JuLIP interface will be retired.
+
+## Loading Training Data
+
+With AtomsBase you can use [AtomsIO](https://github.com/mfherbst/AtomsIO.jl) to load training data
+
+```julia
+using AtomsIO
+
+data = load_trajectory("path to training data")
+```
+
+You can also use ExtXYZ directly (exported by ACEpotentials) to load training data
+
+```julia
+data = ExtXYZ.load("path to training data")
+```
+
+To use AtomsBase with the examples you can load the training data in AtomsBase format by giving a keyword argument `atoms_base=true`
+
+```julia
+data, _, meta = ACEpotentials.example_dataset("TiAl_tutorial"; atoms_base=true)
+```
+
+## Training with AtomsBase
+
+There are no changes to the training methods when using AtomsBase. You only need to have the training data in AtomsBase format for the training to work. Here is the `acemodel` style training tutorial using AtomsBase
+
+```@example
+using ACEpotentials
+
+
+data, _, meta = ACEpotentials.example_dataset("TiAl_tutorial"; atoms_base=true)
+
+model = acemodel(
+    elements = [:Ti, :Al],
+	order = 3,
+	totaldegree = 6,
+	rcut = 5.5,
+	Eref = [:Ti => -1586.0195, :Al => -105.5954]
+)
+@show length(model.basis);
+
+
+weights = Dict(
+    "FLD_TiAl" => Dict("E" => 60.0, "F" => 1.0 , "V" => 1.0 ),
+    "TiAl_T5000" => Dict("E" => 5.0, "F" => 1.0 , "V" => 1.0 )
+);
+
+solver = ACEfit.LSQR(damp = 1e-2, atol = 1e-6);
+data_train = data[1:5:end]
+P = smoothness_prior(model; p = 4) 
+
+acefit!(model, data_train; solver=solver, weights=weights, prior = P);
+
+@info("Training Error Table")
+ACEpotentials.linear_errors(data_train, model; weights=weights);
+```
+
+### Weights with AtomsBase
+
+With AtomsBase weights are stored in AtomsBase structures and there is no need to create `AtomsData` structures. Weight for a structure can be accessed with keyword `:weight`
+
+```julia
+# return weight for a structure 
+data_point[:weight]
```

Comment:
please explain how this per-structure weight modifies the E, F, V weights.

Reply:
done in documentation